CMakeLists.txt

```diff
@@ -22,15 +24,33 @@ set(tinyobjloader-examples-objsticher
 	${TINYOBJLOADEREXAMPLES_DIR}/obj_sticher/obj_sticher.cc
 	)
 
+#Install destinations
+set(TINYOBJLOADER_CMAKE_DIR lib/cmake)
```

Comment:
tinyobjloader is header-only library, thus I'm not sure it is better to make it packageable for distros. 
Anyway, I'll accept a PR( @mwoehlke-kitware , please send a PR of `LIB_SUFFIX`)